projects/components/src/color-picker/color-picker.component.ts

```diff
@@ -49,6 +49,9 @@ export class ColorPickerComponent implements ControlValueAccessor, OnChanges {
   @Input()
   public disabled?: boolean = false;
 
+  @Input()
+  public noDeselectMode?: boolean = false;
```

Comment:
should we call it `disableColorDeselection` or `disableDeselection`?

Reply:
And add a comment here if possible

Reply:
yeah better name, change coming

Reply:
how is it different from `disabled`?

Reply:
It still allows user to select between the options, just not de-selecting the currently selected one